### PR TITLE
test: [M3-6611] - Add new assertions for linode backup tests

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝

The cypress/e2e/core/linodes/backup-linode.spec.ts have a few shortcomings that should be addressed:

- Navigate to the "Backups" tab of a Linode details page using the UI rather than navigating directly to "/linodes/:id/backups"
- Add test/assertions for canceling Linode backups
- Add test/assertions for error flow when attempting to re-enable Linode backups after canceling (users cannot re-enable backups without first waiting 24 hrs)

## Changes  🔄

List any change(s) relevant to the reviewer.

- Add new assertions for linode backup tests

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/linodes/backup-linode.spec.ts"
```

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]
Clean! 🧼 Thanks Azure!

Edit: You're welcome to disregard the test failures but I think all of those are fixed in develop now -- shout out to @bnussman-akamai for the machine images fixes!

File: packages/manager/.changeset/pr-11326-tests-1732566960471.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Tests
+---
+
+Add new assertions for linode backup tests ([#11326](https://github.com/linode/manager/pull/11326))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-11326-tests-1732566960471.md:5]:
```suggestion
Add new assertions for linode backup Cypress tests ([#11326](https://github.com/linode/manager/pull/11326))
```
Optional nit for clarity

**Review by mjac0bs — APPROVED** [review]
Thanks Azure -  confirmed the changes to the flow make sense and spec passes locally.

image

**Review by pmakode-akamai — APPROVED** [review]
Confirmed all tests pass ✅ thank you!
Screenshot 2024-11-26 at 10 01 11 PM

**Pull request merged**